x/evmutil/keeper/keeper.go

```diff
@@ -40,9 +45,15 @@ func NewKeeper(
 		paramSubspace: params,
 		bankKeeper:    bk,
 		accountKeeper: ak,
+		authority:     authority,
 	}
 }
 
+// GetAuthority returns the x/evmutil module's authority.
```

Comment:
is this message used anywhere? Typically for golang you want to avoid java styler getter and setters and only interface the behavior of the desired type...e.g. there might be a method on the interface or concrete type "IsValidAuthority" to allow callers to let the keeper determine validity without exposing its state. Or if the state does need to get exposed, that can more directly be stored in a public field on the struct that can be accessed directly